docs/core/testing/unit-testing-fsharp-with-dotnet-test.md

```diff
@@ -25,7 +25,7 @@ Inside the solution directory, create a *MathService* directory. The directory a
     /MathService
 ```
 
-Make *MathService* the current directory and run [`dotnet new classlib -lang F#`](../tools/dotnet-new.md) to create the source project.  You'll create a failing implementation of the math service:
+Make *MathService* the current directory and run [`dotnet new classlib -lang "F#"`](../tools/dotnet-new.md) to create the source project.  You'll create a failing implementation of the math service:
```

Comment:
The link doesn't render as code even with the backticks. I'd suggest:
```suggestion
Make *MathService* the current directory, and run `dotnet new classlib -lang "F#"` to create the source project. You'll create a failing implementation of the math service:
```
And then add a See also link to `[dotnet new](../tools/dotnet-new.md)`. This problem occurs throughout this article so I'd update all the code example links in this way.